### DISCUSSION TITLE
Apakah Ada Input Field Panel buat Menambahkan Input Panel?

### DISCUSSION THREAD

**Discussion by lokersumatera** [mecha-cms/mecha#200] [Q&A] [answered]
Kang, apakah ada opsi field buat menambahkan sebuah input panel di mecha?

misalnya saya ingin mengupload beberapa gambar. Secara default saya membuat 3 input panel upload gambar. Dan pada custom panel saya terkadang ingin mengupload lebih dari 3 gambar. jadi dengan menekan `[add panel]` nanti akan menambahkan input panel upload gambar ke 4, ke 5, dst.

Apakah ada metode seperti itu di mecha kang? atau metode serupa?

```
gambar 1 ; [upload gambar]
gambar 2 ; [upload gambar]
gambar 3 ; [upload gambar]
tambah panel: [add panel]
```

Sebenarnya saya ingin membuat slider, untuk sekarang saya masih menggunakan cara manual. menggunakan `field/content`,

**Comment by taufik-nurrohman** [comment, marked as answer]
Belum ada, tapi mas bisa _request_ di repositori [@mecha-cms/x.panel.extra](https://github.com/mecha-cms/x.panel.extra). Paling gampang sebenarnya kalau ini diintegrasikan ke ekstensi [@mecha-cms/x.panel.image](https://github.com/mecha-cms/x.panel.image) supaya bisa menerima seleksi beberapa gambar sekaligus.

**Reply by lokersumatera** [reply]
iya kang, tolong follow up ya kang, biar bisa upload beberapa gambar sekaligus
